exonum-java-binding/core/src/main/proto/testkit.proto

```diff
@@ -0,0 +1,27 @@
+// Copyright 2019 The Exonum Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package exonum.testkit;
+
+option java_package = "com.exonum.binding.testkit";
+option java_outer_classname = "TestKitProtos";
+
+import "runtime.proto";
+
+message TestKitServiceInstances {
+    repeated exonum.runtime.ArtifactSpec artifact_specs = 1;
```

Comment:
(Nit) May add short comments to fields explaining _which_ artifacts and services (artifacts to _deploy_, services to _start_ — and both — on blockchain instantiation)